icaruscode/CRT/CRTUtils/CRTHitRecoAlg.cc

```diff
@@ -364,19 +367,28 @@ sbn::crt::CRTHit CRTHitRecoAlg::MakeTopHit(art::Ptr<CRTData> data){
     
     auto const& adsGeo = adGeo.SensitiveVolume(adsid_max); //trigger strip
     uint64_t thit = data->fTs0;
+    uint64_t thit1 = data->fTs1;
 
-    if(adsid_max<8)
-        thit -= hitpos.Z()*fPropDelay;
-    else
-        thit -= hitpos.X()*fPropDelay;
-
+    if(adsid_max<8){
+        thit -= (uint64_t)round(abs((92+hitpos.X())*fPropDelay));
+        thit1 -= (uint64_t)round(abs((92+hitpos.X())*fPropDelay));
+	thit -= 11; //Correction for 11 ns signal cable from SiPM to FEB
```

Comment:
Thanks for putting this together.
Please add a fhicl parameter for 11 ns. It is always better to remove any hardcoded information. Same comments for L381 and L382.

Reply:
@biswaranjan91 added the SiPM to FEB propagation as crtsimproducer.fcl parameter